workflows/diagnostics/fv3net/diagnostics/_shared/constants.py

```diff
@@ -28,3 +26,5 @@ class DiagArg:
     verification: xr.Dataset
     grid: xr.Dataset
     delp: Optional[xr.DataArray] = None
+    horizontal_dims: List[str] = field(default_factory=lambda: ["x", "y", "tile"])
+    vertical_dim: str = "z"
```

Comment:
Is the plan to use `z` or `lev` for the vertical dimension? There are still some places where `z` is hard-coded. Probably better to leave it hard-coded everywhere (fine with me) or make sure it is actually configurable.

Reply:
The plan is to use `z`, retained `VERTICAL_DIM="z"` and reused it here.